src/extras/core/Gyroscope.js

```diff
@@ -1,65 +1,65 @@
-/**

```

Comment:
Do I need to revert them??

because https://github.com/mrdoob/three.js/pull/6859?w=1 seems clean to me